test/framework/pipelineRun_helper.go

```diff
@@ -38,7 +38,9 @@ func executePipelineRunTests(ctx context.Context, t *testing.T, testPlans ...Tes
 			name :=
 				fmt.Sprintf("%s_%d", getTestPlanName(testPlan), i)
 			ctx = SetTestName(ctx, name)
-			pipelineTest := testPlan.TestBuilder(tnn)
+			runID := &api.CustomJSON{map[string]string{"name": name}}
```

Comment:
```suggestion
			runID := &api.CustomJSON{map[string]string{"buildId": name}}
```

Since (at least in our internal case) the `runID` consists of fields `buildId`, `jobId` and `realmId` (all strings) we should not confuse Elasticsearch with new fields. Especially since we run the tests also on the productive systems.